Header header
uint32 imageWidth
uint32 imageHeight
uint32 blobCount
Blob[] blobs
